apps/ejabberd/src/pubsub_migrate.erl

```diff
@@ -0,0 +1,443 @@
+%%%----------------------------------------------------------------------
+%%% File    : pubsub_migrate.erl
```

Comment:
I think we can remove this module completely. There is nothing to migrate from as for now.